pnpm-lock.yaml

```diff
@@ -1162,84 +1162,6 @@ importers:
       stylelint-scss: 4.6.0_stylelint@15.10.1
       typescript: 4.9.5
 
-  enterprise/packages/auth:
```

Comment:
did you pull the changes from the `submodules` because it seems like you didn't and that's why this lock file was updated, [how to do ref](https://www.notion.so/novuhq/Git-Submodules-243ac16a181c4bb49d51241c63bc9226?pvs=4#0bc316fbe3474e53a9fa780af9feaa90)